docs/guides/dev-setup/dev-windows.md

```diff
@@ -0,0 +1,367 @@
+    </figure>
+
+    3. Under "System Variables", selected "Path" and then "Edit..."
+
+    <figure markdown="span">
+      ![selecting path and then clicking edit on system variables](../../images/dev-setup/windows/env-variables-3.png){ width="300" }
+    </figure>
+
+    4. Click "New".
+
+    <figure markdown="span">
+      ![selecting new on environment variables window](../../images/dev-setup/windows/env-variables-4.png){ width="300" }
+    </figure>
+
+    5. Type in the location of your MySQL bin folder. For example, if you followed our XAMPP guide, this will be located `C:\xampp\mysql\bin`. Click OK to save.
+
+    <figure markdown="span">
+      ![entering mysql path into environment variables window](../../images/dev-setup/windows/env-variables-5.png){ width="300" }
+    </figure>
+
+    6. Restart Visual Studio Code as well as any other command prompts you have open for the changes to reflect.
+
+11. Now for some Lorekeeper-specific commands. Run `php artisan add-site-settings` to populate the database with the site settings. It should look like this when completed:
+
+<figure markdown="span">
+  ![alt text](../../images/dev-setup/windows/running-lk-12.png){ width="600" }
+</figure>
+
+12. Then run `php artisan add-text-pages`. This will add the default text pages for LK:
+
+<figure markdown="span">
+  ![alt text](../../images/dev-setup/windows/running-lk-11.png){ width="600" }
+</figure>
+
+13. Next run `php artisan copy-default-images`. This will add the default images for various features on LK:
+
+<figure markdown="span">
+  ![alt text](../../images/dev-setup/windows/running-lk-16.png){ width="600" }
+</figure>
+
+14. Finally, we will run the command `php artisan setup-admin-user`. This will create user #1, the default admin account. The email and details you give it do not need to be real, but make them something you'll remember easily.
+
+- `Proceed to create account with this information? (yes/no)` Answer **yes**.
+- `Are you on a local/testing instance and not a live site? (yes/no)` Answer **yes**.
+- `Would you like to mark your email address as verified and enter an alias now? (yes/no)` Answer **yes**.
+
+The important details to mark as `yes` have been highlighted in red.
+
+<figure markdown="span">
+  ![alt text](../../images/dev-setup/windows/running-lk-13.png){ width="600" }
+</figure>
+
+15. Now, go to `http://localhost` in your browser. You can also access this by pressing the "Admin" button next to "Apache" in XAMPP.
+
+<figure markdown="span">
+  ![alt text](../../images/dev-setup/windows/running-lk-15.png){ width="600" }
+</figure>
+
+## Setup Complete
+
+You have finished installing a local copy of Lorekeeper. You can login as the admin account you just set up, and begin trying out the different features.
+
+<figure markdown="span">
+  ![alt text](../../images/dev-setup/windows/running-lk-14.png){ width="600" }
+</figure>
```

Comment:
I think it would be great to have the end of this page link back to the main setup index the same way the other one does